app/angular/src/client/preview/angular-beta/ComputesTemplateFromComponent.ts

```diff
@@ -52,3 +53,66 @@ export const computesTemplateFromComponent = (
 
   return `<${ngComponentMetadata.selector}${templateInputs}${templateOutputs}>${innerTemplate}</${ngComponentMetadata.selector}>`;
 };
+
+const toAngularProperty = ({
+  propertyName,
+  value,
+  argType,
+}: {
+  propertyName: string;
+  value: any;
+  argType: ArgType;
```

Comment:
```suggestion
  argType?: ArgType;
```

could be undefined si `ArgTypes` is undefined to, no ?